features/unicode-range.yml

```diff
@@ -0,0 +1,7 @@
+name: unicode-range
+description: The `unicode-range` CSS descriptor sets the characters to be used from a font face. If the characters aren't present on the page, the font is not downloaded.
```

Comment:
Non-blocking:
- "sets" doesn't seem right, maybe "specifies"?
- could be more specific about the descriptor contents, eg "from a font face, using Unicode codepoints individually, in a range, or wildcard match"

Reply:
Thanks, I like these suggestions!